File: uri/dart/1009.dart
import 'dart:io';

void main() {

  String name = stdin.readLineSync();
  double fixedSalary = double.parse(stdin.readLineSync());
  double sales = double.parse(stdin.readLineSync());

  double salary = fixedSalary + 0.15 * sales;

  print('TOTAL = R\$ ${salary.toStringAsFixed(2)}');

}

File: uri/go/1009.go
package main

import (
	"fmt"
)

func main() {

	var name string
	var salary float64
	var sales float64

	fmt.Scanf("%s\n%f\n%f", &name, &salary, &sales)

	salary += 0.15 * sales
	fmt.Printf("TOTAL = R$ %.2f\n", salary) 
	
}